algos/test/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="" >
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="generator" content="pandoc" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
+  <meta name="author" content="The Algorithmist" />
+      <meta name="keywords" content="algorithms,tutorial,visualization" />
+    <title>Foo bar</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway|Arimo" type="text/css" />
+  <link rel="stylesheet" href="css/styles.css" type="text/css" />
+  <script type="module" type="text/javascript">
+    import katex from 'https://cdn.jsdelivr.net/npm/katex@0.10.1/dist/katex.mjs';
+    ...
```

Comment:
What do the dots mean?

Reply:
This is an accidentally committed generated file with an old, buggy version of the template. I'll remove the whole file (applies to all `algos/test/index.html` comments unless otherwise noted).